rfcs/text/0041/asset.yml

```
@@ -0,0 +1,147 @@
+    example: TBD
+    description: Used primarily for 'Host' entities, the version or year associated with the asset.
+- name: asset.owner
+    level: extend
+    type: keyword
+    example: sourin.paul@elastic.co
+    description: The primary user entity who owns the 'Host' asset
+- name: asset.priority
+    level: extend
+    type: keyword
+    example: Priority 1
+    description: A priority classification for the asset obtained from outside this system, such as from some external CMDB or Directory service.
+- name: asset.criticality
+    level: extend
+    type: keyword
+    example: Critical
+    description: A criticality classification obtained from outside this system, such as from some external CMDB or Directory service.
+- name: asset.business_unit
+    level: extend
+    type: keyword
+    example: Analyst Experience
+    description: Business Unit associated with the asset (user or host).
+- name: asset.costCenter
+    level: extend
+    type: keyword
+    example: Security - Protections
+    description: Cost Center associated with the asset (user or host).
+- name: asset.cost_center_hierarchy
+    level: extend
+    type: keyword
+    example: Engineering
+    description: Additional cost center information associated with the asset (user or host).
+- name: 
+    level: extend
+    type: 
+    example: 
+    description: 
+- name: asset.status
+    level: extend
+    type: keyword
+    example: ACTIVE
+    description: Current status of the asset in the inventory data source.
+- name: asset.last_status_change_date
+    level: extend
+    type: date
+    example: June 5, 2023 @ 18:25:57.000
+    description: The most recent date/time when the asset.status was updated.
+- name: asset.create_date
+    level: extend
+    type: date
+    example: June 5, 2023 @ 18:25:57.001
+    description: For users, it's the hire date. For other assets, it's the in-service date.
+- name: asset.end_date
+    level: extend
+    type: date
+    example: June 5, 2023 @ 18:25:57.002
+    description: For users, it's the termination date. For other assets, it's the out-of-service date.
+- name: asset.first_seen
+    level: extend
+    type: date
+    example: June 5, 2023 @ 18:25:57.003
+    description: The earliest date/time at which this asset was observed.  
+- name: asset.last_seen
+    level: extend
+    type: date
+    example: June 5, 2023 @ 18:25:57.004
+    description: The most recent date/time this asset was observed.  It would remain empty until the asset was observed.
+- name: asset.last_updated
+    level: extend
+    type: date
+    example: June 5, 2023 @ 18:25:57.005
+    description: The most recent date/time this asset was updated in the inventory data source
+- name: asset.serial_number
+    level: extend
+    type: keyword
+    example: C02FG1G1MD6T
+    description: Serial number of the asset
+- name: asset.tags
+    level: extend
+    type: keyword
+    example: watch, mdmaccess
+    description: Tags assigned at the MDM
```

```suggestion
    description: Tags assigned at the MDM
    normalize:
      - array
```

nit: use `normalize: [ array ]` since we're expecting field hold an array of value, similar to other fields like the top-level [`tags`](https://github.com/elastic/ecs/blob/main/schemas/base.yml#L45).